browser/js/common/directives/navbar/navbar.js

```diff
@@ -9,14 +9,46 @@ app.directive('navbar', function ($rootScope, AuthService, AUTH_EVENTS, $state,
 			scope.goToSection = GoToSectionFactory.goToSection;
 			scope.class = 'hide-menu';
 
+			scope.displayWhatIs = true;
+			scope.displayGetInvolved = true;
+			scope.displayResources = true;
+			scope.displayOurTeam = true;
+
+			// This toggle menu is used for the hamburger icon and when to
+			// display the titles of the dropdown menus
 			scope.toggleMenu = function () {
 				if (scope.class == 'hide-menu') {
 					scope.class = 'show-menu';
 				} else {
 					scope.class='hide-menu';
 				}
 			}
-			
+
+			// This toggle function is used for the mobile nav-bar lists
+			scope.toggle = function (val) {
+				if (val == "displayWhatIs") {
+					scope.displayWhatIs = !scope.displayWhatIs;
+					scope.displayGetInvolved = true;
+					scope.displayResources = true;
+					scope.displayOurTeam = true;
```

Comment:
what about something like 
```js
scope.toggle = function (val) {
  const scopeVars = ['displayWhatIs', 'displayGetInvolved', 'displayResources', 'displayOurTeam'];
  scopeVars.forEach(scopeVar => {
    if (scopeVar === val) {
      scope[scopeVar] = !scope[scopeVar];
    } else {
      scope[scopeVar] = true;
    }
  });
}
```
Something like that (not sure if this exact solution would work) could replace all that logic. Does that make sense?